sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsServiceSerializer.cs

```diff
@@ -404,14 +404,14 @@ private static AnalyzeSentimentResult ReadDocumentSentimentResult(JsonElement do
 
         private static DocumentSentiment ReadDocumentSentiment(JsonElement documentElement, string scoresElementName)
         {
-            SentimentLabel sentiment = default;
+            DocumentSentimentLabel sentiment = default;
             double positiveScore = default;
             double neutralScore = default;
             double negativeScore = default;
 
             if (documentElement.TryGetProperty("sentiment", out JsonElement sentimentValue))
             {
-                sentiment = (SentimentLabel)Enum.Parse(typeof(SentimentLabel), sentimentValue.ToString(), ignoreCase: true);
+                sentiment = (DocumentSentimentLabel)Enum.Parse(typeof(DocumentSentimentLabel), sentimentValue.ToString(), ignoreCase: true);
```

Comment:
Is it safe for me to assume the `sentimentValue` is a text label rather than the integer value of the member?

Reply:
correct.